### DISCUSSION TITLE
Can I update from Next 13 to 14?

### DISCUSSION THREAD

**Discussion by locbqvmo** [vercel/next.js#67483] [Help]
### Summary

My current codebase is using NextJS 13, it was inherited by the previous team. I do a round update packages every 6 months, but haven't update to Next 14 yet. 
If I still use Page Router, can I update to Next 14, any break changes that I need to be aware of?

**Comment by Choi-Jiwon-38** [comment]
Next.js still supports page router in its latest version. However, migrating to the App Router may be necessary to utilize the newest features.

https://nextjs.org/docs/pages

**Reply by VarnikaManvik** [reply]
Hello Team,

I need assistance in updating my Next.js application from version 12.2.3 to the latest or highest version. My project currently uses React and React-DOM version 17.0.1.

However, when I attempt the update, the application does not function correctly due to numerous breaking changes and code breakages. My project is structured using the traditional page-based approach, not the latest app routing structure.

My goal is to update Next.js without making significant changes to the codebase or file structure. Any guidance or suggestions on how to achieve this would be greatly appreciated.

Thank you!

(Question Link : https://github.com/vercel/next.js/discussions/67906)

**Comment by little077** [comment]
I understand that it's completely possible, because my project version has just been upgraded from 12 to 14 and only modified the Link and Script components, so there isn't much content left. If you were in version 13, it should be much simpler（ https://nextjs.org/docs/pages/building-your-application/upgrading/version-14 ）
